app/FreshRSS.php

```diff
@@ -74,10 +74,10 @@ private static function initAuth(): void {
 			if (!FreshRSS_Context::hasSystemConf() || !(FreshRSS_Auth::isCsrfOk() ||
 				(Minz_Request::controllerName() === 'auth' && Minz_Request::actionName() === 'login') ||
 				(Minz_Request::controllerName() === 'user' && Minz_Request::actionName() === 'create' && !FreshRSS_Auth::hasAccess('admin')) ||
-				(Minz_Request::controllerName() === 'feed' && Minz_Request::actionName() === 'actualize'
-					&& FreshRSS_Context::systemConf()->allow_anonymous_refresh) ||
-				(Minz_Request::controllerName() === 'javascript' && Minz_Request::actionName() === 'actualize'
-					&& FreshRSS_Context::systemConf()->allow_anonymous)
+				(Minz_Request::controllerName() === 'feed' && Minz_Request::actionName() === 'actualize' &&
```

Comment:
This is less immediately obvious, but probably no big deal.

Reply:
This one is a rule that asks to have boolean operators either at the end of the line or at the beginning, but consistently within the same statement

Reply:
Fwiw, I think

bla
&& bla

is clearer than

bla &&
bla

But other people have a different opinion.

Reply:
Yes, both are allowed, but not:

```
aa &&
bb
&& cc
```

Reply:
But there was no mixture here.

Reply:
Yes, there was a mixture, with trailing `||` on the first line, for instance